docs/resources/workload_identity_provider.md

```diff
@@ -10,7 +10,51 @@ order: 26
 
 Manages the state of workload identity providers in Scalr.
 
+## Example Usage
 
+### Github Actions
+
+```terraform
+resource "scalr_workload_identity_provider" "github" {
+  name              = "github-actions"
+  url               = "https://token.actions.githubusercontent.com"
+  allowed_audiences = ["scalr-github-actions"]
+}
+
+resource "scalr_assume_service_account_policy" "ga-scalr-staging" {
+  name                     = "ga-scalr-staging"
+  service_account_id       = scalr_service_account.staging.id
+  provider_id              = scalr_workload_identity_provider.github.id
+  maximum_session_duration = 7200
+  claim_condition {
+    claim    = "sub"
+    value    = "repo:GithubOrganization/repository:environment:staging"
+    operator = "startswith"
+  }
+}
```

Comment:
I think this resource is redundant here, I'd just be focused on the `scalr_workload_identity_provider` resource.

And it is more meaningful to add it [here](https://github.com/Scalr/terraform-provider-scalr/pull/412/files#diff-68cf9c5e893413140d09903842231404966753681e2e2ac8e17af009cc71a645R18-R33)